packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -119,6 +119,19 @@ class WaitUntilNoTransientCallbacks extends Command {
   String get kind => 'waitUntilNoTransientCallbacks';
 }
 
+/// A Flutter Driver command that waits until the frame is synced.
+class WaitUntilFrameSync extends Command {
```

Comment:
`WaitUntilFrameSync` makes it sounds like it's synchronously waiting.  Maybe call it `WaitUntilNoScheduledFrame` or something similar?

Reply:
Oops.. Merged a couple of mins ago. I'll follow up in a separate commit if needed.

This method checks both the transient callbacks, and the hasScheduledFrame value (if we discover more states/conditions we should check, I'm planning to add to this method). And since there's already a concept of "frame sync" in Driver, and this method is defined to check that state before returning,`WaitUntilFrameSync` sounds fine to me, but let me know if you'd like to rename it.

Reply:
No worries.  Yeah I think it's worth changing since "sync" at the end of a method name in Dart means "do this synchronously" (e.g. `completeSync()`, `addSync()`, `readAsBytesSync()`, etc.).

Reply:
Thanks for explaining! I'll rename it in another commit.